index.html

```diff
@@ -8,6 +8,7 @@
     <title>Sanam's Protfolio</title>

```

Comment:
According the importance of element, I think the image I added below describe the headings hierarchy better. You used h2 element for the different elements with different importance and sizes. Depending on the Figma font-size, that would be a better order. Kindly fix them.
![Screenshot 2023-04-11 at 23-20-18 Sanam's Protfolio](https://user-images.githubusercontent.com/25887253/231280982-09e29366-c371-469f-b4b5-b333ab4ba908.png)

Reply:
-Rectified `headings` hierarchy with respect to the importance  of the headings per section